<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="dummyWidget1" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="outerTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="sourcesTab">
          <attribute name="title">
           <string>Source Files</string>
          </attribute>
          <layout class="QVBoxLayout" name="sourceFilesLayout">
          </layout>
         </widget>
         <widget class="QWidget" name="outputOptionsTab">
          <attribute name="title">
           <string>Output Options</string>
          </attribute>
          <layout class="QHBoxLayout" name="yyy1">
           <item>
            <layout class="QVBoxLayout" name="outputOptionsLHS">
             <item>
              <widget class="QGroupBox" name="outputFormatBox">
               <property name="title">
                <string>Output Format</string>
               </property>
               <widget class="QWidget" name="layoutWidget_33">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>40</y>
                  <width>383</width>
                  <height>88</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_12">
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="outputAVI">
                   <property name="text">
                    <string>AVI</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QRadioButton" name="outputSER">
                   <property name="text">
                    <string>SER</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QRadioButton" name="outputBMP">
                   <property name="text">
                    <string>BMP</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="outputTIFF">
                   <property name="text">
                    <string>TIFF</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QRadioButton" name="outputFITS">
                   <property name="text">
                    <string>FITS</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QRadioButton" name="outputPNG">
                   <property name="text">
                    <string>PNG</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" colspan="3">
                  <widget class="QRadioButton" name="outputAnimGIF">
                   <property name="text">
                    <string>Animated GIF (8-bit mono/colour only)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="outputDirBox">
               <property name="title">
                <string>Output Directory</string>
               </property>
               <layout class="QVBoxLayout" name="yyyz6">
                <item>
                 <widget class="QLineEdit" name="lineEdit_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_19">
                  <item>
                   <widget class="QPushButton" name="useDefaultOutputDir">
                    <property name="text">
                     <string>Use default</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="selectOutputDir">
                    <property name="text">
                     <string>Select directory</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QGroupBox" name="outputSubdirBox">
                  <property name="title">
                   <string>Output Subdirectory</string>
                  </property>
                  <layout class="QVBoxLayout" name="yyyx6">
                   <item>
                    <widget class="QCheckBox" name="createOutputSubdir">
                     <property name="text">
                      <string>Create subdirectory</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_20">
                     <item>
                      <widget class="QLabel" name="subdirNameLabel">
                       <property name="text">
                        <string>Subdirectory name</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="outputSubdirName">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="appendDateTime">
                     <property name="text">
                      <string>Append date &amp; time</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="prefixSuffixBox">
               <property name="title">
                <string>Output Filename Prefix/Suffix</string>
               </property>
               <widget class="QWidget" name="layoutWidget_34">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>20</y>
                  <width>200</width>
                  <height>62</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_13">
                 <item row="0" column="0">
                  <widget class="QLabel" name="outputFilePrefixLabel">
                   <property name="text">
                    <string>Prefix</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="outputFilePrefix"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="outputFileSuffixLabel">
                   <property name="text">
                    <string>Suffix</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="outputFileSuffix"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="channelSplittingBox">
               <property name="title">
                <string>Colour (RGB) Channel Splitting</string>
               </property>
               <widget class="QCheckBox" name="splitRGB">
                <property name="geometry">
                 <rect>
                  <x>30</x>
                  <y>50</y>
                  <width>171</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Split RGB channels</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="outputOptionsRHS">
             <item>
              <widget class="QGroupBox" name="outputFrameDroppingBox">
               <property name="title">
                <string>Output Frame Dropping</string>
               </property>
               <widget class="QWidget" name="layoutWidget_35">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>40</y>
                  <width>341</width>
                  <height>27</height>
                 </rect>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_21">
                 <item>
                  <widget class="QSpinBox" name="framesToDrop"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="droppedLabel">
                   <property name="text">
                    <string>Frames to drop after each frame is output</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="aviFileOptionsBox">
               <property name="title">
                <string>AVI File Options</string>
               </property>
               <widget class="QWidget" name="layoutWidget_36">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>21</y>
                  <width>441</width>
                  <height>181</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_16">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_24">
                   <item>
                    <widget class="QComboBox" name="outputCodec"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="codecLabel_2">
                     <property name="text">
                      <string>AVI codec</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="showCodec">
                     <property name="text">
                      <string>n/a</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_22">
                   <item>
                    <widget class="QLabel" name="outputQualityLabel">
                     <property name="text">
                      <string>Quality (1: lowest, 10: best)</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="spinBox"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_23">
                   <item>
                    <widget class="QCheckBox" name="oldFormatAVI">
                     <property name="text">
                      <string>Generate old format AVI file</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="maxAVISize">
                     <item>
                      <property name="text">
                       <string>4GB maximum size</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>2GB maximum size</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="safeSize">
                   <property name="text">
                    <string>Modify AVI frame width to safe size (DIB codec only)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="matchInputFrameRate">
                   <property name="text">
                    <string>Match input frame rate where possible</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_25">
                   <item>
                    <widget class="QSpinBox" name="defaultFrameRate"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="defaultFrameRateLabel_2">
                     <property name="text">
                      <string>Frame rate to use otherwise</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="winJuposBox">
               <property name="title">
                <string>WinJUPOS Compatible Filenames</string>
               </property>
               <widget class="QWidget" name="layoutWidget_37">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>30</y>
                  <width>365</width>
                  <height>53</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_17">
                 <item>
                  <widget class="QCheckBox" name="winjuposNames">
                   <property name="text">
                    <string>Generate winJUPOS-compatible names</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_26">
                   <item>
                    <widget class="QLabel" name="filenameTimeLabel">
                     <property name="text">
                      <string>Time to use for filename</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="filenameTime">
                     <item>
                      <property name="text">
                       <string>Output frames mid-point time</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Earlliest output frame time</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>First output frame time</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="doProcessingTab">
          <attribute name="title">
           <string>Do Processing</string>
          </attribute>
          <layout class="QHBoxLayout" name="ww1">
           <item>
            <layout class="QVBoxLayout" name="ww2">
             <item>
              <widget class="QGroupBox" name="controlBox">
               <property name="title">
                <string>Control</string>
               </property>
               <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>60</y>
                  <width>258</width>
                  <height>64</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_18">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_27">
                   <item>
                    <widget class="QPushButton" name="startProcessing">
                     <property name="text">
                      <string>Start processing</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="cancelProcessing">
                     <property name="text">
                      <string>Cancel processing</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="openOutputDir">
                   <property name="text">
                    <string>Open output directory</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="statusBox">
               <property name="title">
                <string>Status</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_19">
                <item>
                 <widget class="QProgressBar" name="progressBar">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="value">
                   <number>24</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="ww5">
               <property name="title">
                <string>Processing</string>
               </property>
               <widget class="QWidget" name="layoutWidget_40">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>30</y>
                  <width>441</width>
                  <height>100</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_20">
                 <item>
                  <widget class="QLabel" name="processingFilename">
                   <property name="text">
                    <string>File: %1</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_29">
                   <item>
                    <widget class="QLabel" name="fileOf">
                     <property name="text">
                      <string>File %1 of %2</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QProgressBar" name="progressBar_2">
                     <property name="value">
                      <number>24</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="processingFrames">
                   <property name="text">
                    <string>Processing %1 frames</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_28">
                   <item>
                    <widget class="QLabel" name="framesPercentage">
                     <property name="text">
                      <string>%1%</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QProgressBar" name="progressBar_3">
                     <property name="value">
                      <number>24</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="ww6">
             <item>
              <widget class="QGroupBox" name="doProcessingRHS">
               <property name="title">
                <string>Output</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <widget class="QWidget" name="layoutWidget_38">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget_39">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddSourceFiles"/>
    <addaction name="actionRemoveSourceFiles"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionResetOptions"/>
    <addaction name="actionUndoResetOptions"/>
    <addaction name="separator"/>
    <addaction name="actionSaveOptions"/>
    <addaction name="actionLoadOptions"/>
    <addaction name="separator"/>
    <addaction name="actionSaveRejectedFrames"/>
    <addaction name="actionUseRAMBuffers"/>
   </widget>
   <widget class="QMenu" name="menuProcessing">
    <property name="title">
     <string>Processing</string>
    </property>
    <addaction name="actionTestOptions"/>
    <addaction name="actionStartProcessing"/>
    <addaction name="actionCancelProcessing"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuProcessing"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAddSourceFiles">
   <property name="text">
    <string>Add Source Files</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionRemoveSourceFiles">
   <property name="text">
    <string>Remove Source Files</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionResetOptions">
   <property name="text">
    <string>Reset Options</string>
   </property>
  </action>
  <action name="actionUndoResetOptions">
   <property name="text">
    <string>Undo Reset Options</string>
   </property>
  </action>
  <action name="actionSaveOptions">
   <property name="text">
    <string>Save Options</string>
   </property>
  </action>
  <action name="actionLoadOptions">
   <property name="text">
    <string>Load Options</string>
   </property>
  </action>
  <action name="actionSaveRejectedFrames">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save Rejected Frames</string>
   </property>
  </action>
  <action name="actionUseRAMBuffers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use RAM Buffers For Quality Sorting</string>
   </property>
  </action>
  <action name="actionTestOptions">
   <property name="text">
    <string>Test Options</string>
   </property>
  </action>
  <action name="actionStartProcessing">
   <property name="text">
    <string>Start Processing</string>
   </property>
  </action>
  <action name="actionCancelProcessing">
   <property name="text">
    <string>Cancel Processing</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
